lib/mutant/isolation/none.rb

```diff
@@ -12,12 +12,25 @@ class None < self
       #
       # @return [Result]
       #
-      # ignore :reek:UtilityFunction
+      # rubocop:disable Lint/SuppressedException
```

Comment:
Rubocop has something to learn here. `=> expression` is an assignment. Unparser also has to learn this to reproduce lvar scope specific syntax.